### PR TITLE
Cookie rewrites

### PR DESCRIPTION
**Pull request by xapiksw**
Implemented reading `action_type` from Cookies instead of page response.

### CODE REVIEW

**Comment by alfredsgenkins** [comment]
Does this PR has a dependency on other composer module?

**Comment by xapiksw** [comment]
This PR depends on https://github.com/scandipwa/route717/pull/3 and https://github.com/scandipwa/scandipwa-base/pull/38

File: src/app/util/Cookie/Cookie.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+        if (result === undefined) {
+            return null;
+        }
+        const cookie = result.trimStart();
+
+        return cookie.substring(nameString.length, cookie.length);
+    },
+
+    steal(name) {
```
Comment (alfredsgenkins) [inline src/app/util/Cookie/Cookie.js]: Great naming, haha 😁

**Comment by alfredsgenkins** [comment]
Closing, despite good implentation and initial idea this turned out to be redundant complication.

**Pull request closed without merging**